lib/assets/index.js

```diff
@@ -1,5 +1,18 @@
 module.exports = {
-  'icon-32': __dirname + '/icon-32.png',
-  'icon-paused-32': __dirname + '/icon-paused-32.png',
-  'icon-playing-32': __dirname + '/icon-playing-32.png'
+	darwin: {
+		icon: __dirname + '/darwin-icon.png',
+		playing: __dirname + '/darwin-icon-playing.png',
+		paused: __dirname + '/darwin-icon-paused.png'
+	},
+	win32: {
+		// Can use darwin icons, because sizes are the same
+		icon: __dirname + '/darwin-icon.png',
+		playing: __dirname + '/darwin-icon-playing.png',
+		paused: __dirname + '/darwin-icon-paused.png'
+	},
+	any: {
+		icon: __dirname + '/icon.png',
+		playing: __dirname + '/icon-playing.png',
+		paused: __dirname + '/icon-paused.png'
+	}
```

Comment:
That being said, this is still flawed since we might want different sizes based on tray for different platforms so we really need to cross that bridge first =/ (e.g. we might introduced `tray-paused`)